src/group_theory/group_action/basic.lean

```diff
@@ -294,20 +294,11 @@ variables [group α] [mul_action α β]
 
 open quotient_group
 
-/-- Action on left cosets. -/
-@[to_additive "Action on left cosets."]
-def mul_left_cosets (H : subgroup α)
-  (x : α) (y : α ⧸ H) : α ⧸ H :=
-quotient.lift_on' y (λ y, quotient_group.mk ((x : α) * y))
-  (λ a b (hab : _ ∈ H), quotient_group.eq.2
-    (by rwa [mul_inv_rev, ← mul_assoc, mul_assoc (a⁻¹), inv_mul_self, mul_one]))
-
 @[to_additive] instance quotient (H : subgroup α) : mul_action α (α ⧸ H) :=
-{ smul := mul_left_cosets H,
-  one_smul := λ a, quotient.induction_on' a (λ a, quotient_group.eq.2
-    (by simp [subgroup.one_mem])),
-  mul_smul := λ x y a, quotient.induction_on' a (λ a, quotient_group.eq.2
-    (by simp [mul_inv_rev, subgroup.one_mem, mul_assoc])) }
+{ smul := λ g, quotient.map' (λ h, g * h)
```

Comment:
```suggestion
{ smul := λ g, quotient.map' ((*) g)
```
Might fix the build error

Reply:
@eric-wieser I'm not sure what's going on with the build error, but the action should be left-multiplication, not right-multiplication.

Reply:
Oh nevermind. `(*) g` means `g` is the first argument of `(*)`.

Reply:
And thanks for fixing the build error!